gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -24,29 +25,29 @@ Source0: <%= download_path %>%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
 Requires: foreman-proxy >= %{foreman_proxy_min_version}
-Requires: ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(release)
 <% for req in spec.required_ruby_version -%>
-Requires: ruby <%= req %>
+Requires: %{?scl_prefix_ruby}ruby <%= req %>
```

Comment:
Could you use the `SCL_PREFIXES` to pull it from the correct SCL? https://github.com/theforeman/foreman-packaging/blob/682a5f1a0803c5aa422c8495fb19f56494058f36/gem2rpm/foreman_plugin.spec.erb#L64

Note it also needs to read these: https://github.com/theforeman/foreman-packaging/blob/682a5f1a0803c5aa422c8495fb19f56494058f36/gem2rpm/foreman_plugin.spec.erb#L6-L9